package.json

```diff
@@ -68,6 +68,7 @@
     "codemirror": "^6.0.0",
     "fast-xml-parser": "^4.0.10",
     "lodash-es": "^4.17.21",
+    "react-dropzone": "^14.2.3",
```

Comment:
@pkulkark IIRC, you need to commit `package-lock.json` as well.

Reply:
@Cup0fCoffee You're right. I committed it now.